kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java

```diff
@@ -18,6 +18,7 @@
 	public static final int INTERNAL_INCONSISTENCY = 4003;
 	public static final int ORDER_NOT_FOUND = 4004;
 	public static final int ACCOUNT_NOT_LOGGED_IN = 4005;
+	public static final int SDK_IS_NOT_STARTED = 4005;
```

Comment:
Did you intentionally made it the same number as ACCOUNT_NOT_LOGGED_IN(4005)?
If not then change, if yes then just merge

Reply:
yes, kind of backward compatibility if someone outside using this const (internally I'm using the new one)